### ISSUE TITLE
Can the .exe be deleted after running it for the first time?

### ISSUE THREAD

**Issue by Nine-Wonders** [UnamSanctam/SilentCryptoMiner#2336]
I ran the exe once just to get things started. Since it's supposed to be persistent across reboots, can I delete the .exe and expect it to continue to work as normal?

**Comment by UnamSanctam** [comment]
Yes if you have "Startup" enabled in the miner then it will install itself in the install location, so the original .exe that was run can be safely deleted.

**Comment by Nine-Wonders** [comment]
Thanks for the answer. One more thing though, I set the Remote ConFig section on and pasted "https://raw.githubusercontent.com/Nine-Wonders/miner/main/monero_config.json" as the link. I thought it was supposed to grab the content from that raw link every 100 mins but it doesn't. Is there something that I'm supposed to provide in that raw link other than just the JSON config? Thanks in advance!

**Comment by UnamSanctam** [comment]
Your config is correct and should work, though I would caution you against using SupportXMR since they will ban you.

**Comment by sea256** [comment]
> Your config is correct and should work, though I would caution you against using SupportXMR since they will ban you.

wow, I didn't know that, they can detect botnet?

**Comment by UnamSanctam** [comment]
Since they run the pool then they know that you have many low hashrate miner connections, at which point they usually ban you for a suspected botnet.

**Comment by UnamSanctam** [comment]
The most common ones that don't ban (as far as we know) are 2miners and nanopool.

**Comment by Alcinzal** [comment]
+ for 2miners. You can even see that the majority of their users mining monero is silent mining: https://xmr.2miners.com/. The whole pools hashrate fluctuates based on the clock, aka how many people are on their computer.
